app/templates/layouts/_base.html

```diff
@@ -48,10 +48,13 @@
     {% endblock head -%}
     <!-- Google Tag Manager -->
     {% if analytics_gtm_id and analytics_gtm_env_id %}
+      <script>
+        dataLayer = [];
+      </script>
       <script nonce="{{ csp_nonce() }}">(function(w,d,s,l,i,e){w['GoogleTagManagerObject']=l;w[l]=w[l]||[];w[l].push({'gtm.start':
       new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
       j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-      "https://www.googletagmanager.com/gtm.js?id=" + i + dl + "&gtm_auth=" + e + "&gtm_preview=env-8&gtm_cookies_win=x";f.parentNode.insertBefore(j,f);
+      "https://www.googletagmanager.com/gtm.js?id=" + i + dl + "&gtm_auth=" + e;f.parentNode.insertBefore(j,f);
```

Comment:
I see what you mean, and I suppose datalayer could be added to the variable too but the GTM id is separate because that variable is per project and not per environment